### ISSUE TITLE
How do I make this theme work with MacVim?

### ISSUE THREAD

**Issue by yamini-krishnamurthy** [dylanaraps/wal.vim#7]
I would really like to switch to MacVim but I don't know how to use wal.vim for MacVim. Is this possible? Cheers

**Comment by dylanaraps** [comment]
wal.vim doesn't support GUI colorschemes but I can add this feature.

**Comment by yamini-krishnamurthy** [comment]
Awesome! Sounds good.

**Comment by annata83** [comment]
This Pluging works with neovim aswell?

**Comment by dylanaraps** [comment]
Yes it does (I use nvim).

**Comment by annata83** [comment]
How are you @dylanaraps? Any idea of why isn't working for me?

**Comment by dylanaraps** [comment]
```vim
Plug 'dylanaraps/wal.vim'
colorscheme wal
```

**Comment by annata83** [comment]
Yes! Done. Still comes with E185: cannot find color scheme 'wal'

**Comment by dylanaraps** [comment]
Is the colorscheme installed correctly? Have you run `:PlugUpdate`?

**Comment by annata83** [comment]
Yes, Plugin correctly installed and updated as well.

**Comment by 11** [comment]
Has anyone found a solution to the `E185` issue for nvim?

**Comment by 4fthawaiian** [comment]
would love to see this in macvim. Looks awesome in terminal, but would fit into my flow much better in macvim.

**Comment by roman-dusek** [comment]
The edited gui version still didnt work for me, but adding `highlight Normal guibg=#2E3440 guifg=#E5E9F0` and `highlight Cursor guibg=#E5E9F0` to ~/.gvimrc finally works
